.well-known/security.txt

```diff
@@ -0,0 +1,5 @@
+Contact: mailto:security@kubernetes.io
+Expires: 2031-01-11T06:30:00.000Z
```

Comment:
The RFC recommends a one-year expiration or less, which seems like a good idea in principle. But if we publish a short expiration and then forget to update it, that's a bad outcome.

Would we like to make a robust way to remind ourselves to update this expiration, and publish a shorter expiration date? Or shall we publish this as-is because we will probably be keeping the email forever anyway?

Reply:
One idea that comes to mind would be a non-blocking CI test for k/website: if the expiration date is coming up in 3 months, then CI signal goes yellow with a warning message saying to ask SIG Security or SRC to review security.txt and update the expiration date.

Reply:
>The RFC recommends a one-year expiration or less, which seems like a good idea in principle. But if we publish a short expiration and then forget to update it, that's a bad outcome.

I agree, but keeping the expiration date more than a year was actually suggested by @sftim and @reylejano here -> https://github.com/kubernetes/website/pull/44487#discussion_r1449021162

This was the reason that i keep it more than a year i.e 5 years .

Reply:
> One idea that comes to mind would be a non-blocking CI test for k/website: if the expiration date is coming up in 3 months, then CI signal goes yellow with a warning message saying to ask SIG Security or SRC to review security.txt and update the expiration date.

We can built it; we have the technology.